ui/packages/catalog-ui-search/src/main/webapp/component/filter-builder/filter-serialization.js

```diff
@@ -55,6 +55,7 @@ const comparatorToCQL = {
   '=': '=',
   '<=': '<=',
   '>=': '>=',
+  RANGE: 'BETWEEN',
```

Comment:
Look into exporting this object that is already made in `filter.view.js` and importing it here instead of redeclaring.